### PR TITLE
added initial models. Added display of models in admin. some small fixes

### PR DESCRIPTION
**Pull request by wolendranh**

### CODE REVIEW

File: rate_counter/rate/models.py [hunk 1]
```diff
@@ -1,5 +1,40 @@
...
+    name = models.CharField(max_length=100)
+    group = models.ForeignKey('StudentGroup', related_name='subjects')
+
+
+class Institute(models.Model):
+    """
+    Institute model representing Institute parsed from University
+    """
+    name = models.CharField(max_length=4)
+
```
Comment (Scandie) [inline rate_counter/rate/models.py:20]: There should be IntegerField or CharField with site ID of Institute for creating URL for data parsing. Something like:
 identifier = models.IntegerField()

File: rate_counter/rate/models.py [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+
+class StudentGroup(models.Model):
+    """
+    Group representing students group in University
+    """
+    user = models.ForeignKey(User, related_name='student_groups')
+    institute = models.ForeignKey(Institute, related_name='groups')
+    name = models.CharField(max_length=15)
```
Comment (Scandie) [inline rate_counter/rate/models.py:28]: Same as in Institute model.

File: rate_counter/rate/models.py (hunk 1 above)
Comment (wolendranh) [inline rate_counter/rate/models.py:20]: @Scandie Do you mean for second parsing? because first time you will just got it from site i guess

Comment (Scandie) [inline rate_counter/rate/models.py:20]: @wolendranh You don't need it to make a list of institutes. But site identifier of institute is needed in URL for creating a list of groups of this institute. And the same with subjects list.

Comment (wolendranh) [inline rate_counter/rate/models.py:20]: @Scandie Can't you do it like this:
1. get list of all instututes with identifier and name from NULP site (parse them)
2. Store it into DB(create objects of Institutes)
3. Iterate over Institutes objects , get their id or what, and use them in script to format your URL?

Comment (Scandie) [inline rate_counter/rate/models.py:20]: @wolendranh  ok, i would try to do it like this. I think branch could be merged.

Comment (wolendranh) [inline rate_counter/rate/models.py:20]: @Scandie I hope I understood you in correct way :) Anyway we can change models anytime we want. If you did a review - you should merge, we will stick to this schema of review. Thanks for review!

**Pull request merged**